From: David Steele <steele@debian.org>
Date: Mon, 10 Oct 2016 21:25:09 -0400
Subject: Add a PATH for called siritasks

gocryptfs was unable to find fusermount. No PATH was being transferred
to the call.
---
 src/siritask.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/siritask.cpp b/src/siritask.cpp
index 7deea9d..8b42336 100644
--- a/src/siritask.cpp
+++ b/src/siritask.cpp
@@ -207,6 +207,7 @@ static cs _cmd( bool create,const siritask::options& opt,
 
 			env.insert( "CRYFS_NO_UPDATE_CHECK","TRUE" ) ;
 			env.insert( "CRYFS_FRONTEND","noninteractive" ) ;
+			env.insert( "PATH","/bin:/usr/bin" ) ;
 
 			return env ;
 
